src/wallet/wallet2.cpp

```diff
@@ -101,8 +101,9 @@ using namespace cryptonote;
 #define CHACHA8_KEY_TAIL 0x8c
 #define CACHE_KEY_TAIL 0x8d
 
-#define UNSIGNED_TX_PREFIX "Electroneum unsigned tx set"
-#define SIGNED_TX_PREFIX "Electroneum signed tx set"
+// Magic number 004 means these payloads are encrypted.
+#define UNSIGNED_TX_PREFIX "Electroneum unsigned tx set \004"
```

Comment:
There shouldn't be a space between "set" and the magic version number, otherwise new txsets will not be loadable by previous versions of the wallet

Reply:
yes